presto-native-execution/presto_cpp/main/PeriodicTaskManager.cpp

```diff
@@ -38,8 +38,10 @@ static constexpr size_t kExchangeSourcePeriodGlobalCounters{
     2'000'000}; // 2 seconds.
 // Every 1 minute we clean old tasks.
 static constexpr size_t kTaskPeriodCleanOldTasks{60'000'000}; // 60 seconds.
-// Every 1 minute we export cache counters.
-static constexpr size_t kCachePeriodGlobalCounters{60'000'000}; // 60 seconds.
+// Every 1 minute we export connector counters.
+static constexpr size_t kConnectorPeriodGlobalCounters{
+    60'000'000}; // 60 seconds.
+static constexpr size_t kCachePeriodGlobalCounters{600'000'000}; // 10 mins.
```

Comment:
Any reason to make it 10 minutes? Will we miss any significant statistics by this time? Running some tests will be helpful here.

Reply:
Removed this period increase part, as some cache counters are being used now to report hit rate etc. where we still need 1m period. I re-titled this PR.

You're right. Too tired yesterday. I was thinking of reducing the frequency of acquiring the wlock on the cache and reusing the locking period for ttl. Turned out 10mins is too long for hit rate.